conda-store-server/conda_store_server/app.py

```diff
@@ -302,21 +302,24 @@ def _default_celery_results_backend(self):
     )
 
     default_uid = Integer(
-        os.getuid(),
+        None if sys.platform == "win32" else os.getuid(),
```

Comment:
I see `sys.platform == "win32"` in many places. Should we have a constant that we create for this? Since much of our code only has a special path for Windows?

Reply:
More I think about it I don't think this is necissary

Reply:
There was a constant in Aaron's original code. Well, two constants, in different files. Since you need to share this between multiple files, you'd have to put it in `__init__` to avoid import issues. But we don't do this for any other platforms, so I decided to simply use `sys.platform`, as everywhere else, to avoid introducing new entities.